generators/src/new-test-model/new-test-model.generator.ts

```diff
@@ -133,123 +133,135 @@ export const newTestModelGenerator: CodeGenerator = {
+      const filePath = join(
+        outputPath,
+        generationType === 'standalone' ? 'src' : '',
+        template.templatePath
+      );
+      await ensureDirectory(filePath);
+      console.log(`Generating file: ${filePath}`);
+      await writeFile(
+        filePath,
+        renderTemplate(template.templateContent, templatesData),
+        { encoding: 'utf-8' }
+      );
+    }
+
+    if (isDraftRequired) {
+      const filteredModelTemplatesData = modelTemplatesData.filter(
         modelTemplatesFilter({
           isPresetExampleRequired,
           isDraftModel: false,
         })
-      )
-      .forEach((template) => {
+      );
+      for (const template of filteredModelTemplatesData) {
         const filePath = join(
           outputPath,
-          generationType === 'standalone' ? 'src' : '',
+          'src',
+          `${modelCodename}-draft`,
           template.templatePath
         );
-        ensureDirectory(filePath);
+        await ensureDirectory(filePath);
         console.log(`Generating file: ${filePath}`);
-        writeFileSync(
+        await writeFile(
           filePath,
-          renderTemplate(template.templateContent, templatesData)
-        );
-      });
-
-    if (isDraftRequired) {
-      modelTemplatesData
-        .filter(
-          modelTemplatesFilter({
+          renderTemplate(template.templateContent, {
+            isDraftModel: true,
             isPresetExampleRequired,
-            isDraftModel: false,
-          })
-        )
-        .forEach((template) => {
-          const filePath = join(
-            outputPath,
-            'src',
-            `${modelCodename}-draft`,
-            template.templatePath
-          );
-          ensureDirectory(filePath);
-          console.log(`Generating file: ${filePath}`);
-          writeFileSync(
-            filePath,
-            renderTemplate(template.templateContent, {
-              isDraftModel: true,
-              isPresetExampleRequired,
-              modelName: `${modelName}Draft`,
-              modelCodename: `${modelCodename}-draft`,
-              graphqlTypePrefix,
-            })
-          );
-        });
+            modelName: `${modelName}Draft`,
+            modelCodename: `${modelCodename}-draft`,
+            graphqlTypePrefix,
+          }),
+          { encoding: 'utf-8' }
+        );
+      }
     }
 
-    // 3. Update the single package preset
+    // 3. Update the standalone package
     if (generationType === 'standalone') {
-      const presetDirectoryPath = join(
-        __dirname,
-        '..',
-        '..',
-        '..',
-        'presets',
-        'all-packages'
+      const standalonePackagePackageJsonPath = join(
+        rootDir,
+        'package',
+        'package.json'
       );
-      // Generate the re-export proxy file
-      writeFileSync(
-        join(presetDirectoryPath, 'src', `${modelCodename}.ts`),
-        `export * from '@commercetools-test-data/${modelCodename}';`
```

Comment:
This has been removed given the changes with the package's structure (we're now copying the source code of the model itself).